addons/survey/views/survey_user_views.xml

```diff
@@ -152,7 +154,7 @@
         <field name="name">survey.user_input.line.view.form</field>
         <field name="model">survey.user_input.line</field>
         <field name="arch" type="xml">
-            <form string="User input line details" create="false">
+            <form string="User input line details" create="false" edit="false">
```

Comment:
Why removing the edition on those? If the user has the rights?

Reply:
Not sure why this comment is resolved?
Maybe we discussed it and I don't remember.

Reply:
Sorry, thought I replied.

It's to prevent changing answers on a survey and messing with the results.